### DISCUSSION TITLE
Question: Is there any possibility to set user/#preferences/ for every new User to HTML, instead of plain text?

### DISCUSSION THREAD

**Discussion by TarikHaci** [modoboa/modoboa#2079] [Q&A]
# Impacted versions

* Distribution: Ubuntu
* Codename: ?
* Arch: 32 Bits
* Database: PostgreSQL

# Question
How to set that every newly created user has 
Default message display mode: HTML
Enable HTML links display: HTML
Default editor : HTML
In the settings of mail.domain.com/user/#preferences/ ?

Thanks in advance.

**Comment by tonioo** [comment]
@TarikHaci There is no way to do this currently.
